kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/TaskService.java

```diff
@@ -129,7 +129,7 @@ public ProcessData dispatchTask(String sourceCode, String targetCode, String que
 		processData.setProcessId(processId);
 		processData.setAnswers(new ArrayList<>());
 
-		String processEntityCode = Prefix.QBE.concat(targetCode.substring(4));
+		String processEntityCode = Prefix.QBE_.concat(targetCode.substring(4));
```

Comment:
This should use CommonUtils.substitutePrefix(targetCode, Prefix.QBE_)

Reply:
Yes, the Prefix class would be a good location for prefix related utilities.